contracts/VaultV3.vy

```diff
@@ -1048,12 +1061,14 @@ def _update_debt(strategy: address, target_debt: uint256, max_loss: uint256) ->
         withdrawable: uint256 = IStrategy(strategy).convertToAssets(
             IStrategy(strategy).maxRedeem(self)
         )
-        assert withdrawable != 0, "nothing to withdraw"
```

Comment:
is this related?

Reply:
I removed the asserts when increasing debt to not revert if it cant be done. So deposits wont revert if auto allocating.

So i also removed the assert with removing debt for continuity of how the function works both ways.

Not sure if it makes sense or not TBH.